README.md

```diff
@@ -19,3 +25,98 @@ Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_do
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
 be found at <https://hexdocs.pm/wanda>.
 
+## Developing Checks
+
+Wanda architecture aims to simplify [testing Checks Executions](#testing-executions) and [adding new ones](#adding-new-checks). 
+
+### Infrastructure
+
+A docker-compose setup is provided to enable seamless experience with the system.
+
+#### Starting a local environment
+Start the required infrastructure (see [docker-compose.checks.yaml](./docker-compose.checks.yaml))
+
+```bash
+$ docker-compose -f docker-compose.checks.yaml up -d
+```
+
+This starts:
+- wanda, the engine
+- a database where execution informations are stored
+- a message broker for infromation exchange between the targets and wanda
+
+Wanda is exposed on port `4000` and the API documentation is available at http://localhost:4000/swaggerui
+
+![OpenAPI Spec](./guides/img/api_spec.png)
```

Comment:
i wouldn't show this image in the README.
we could instead host the swagger UI in the gh pages

Reply:
Alright, image removed. For the rest `defer`.